configure

```diff
@@ -1045,6 +1050,9 @@ sub write_config_h {
   $opts{'optimize'} = 0 if (!exists $opts{'optimize'} || exists $opts{'sanitize'});
 
   my $CFGH = backup_and_open("$buildEnv{'ACE_ROOT'}/ace/config.h");
+  for my $line (@default_configh) {
```

Comment:
It would be good to have a new command-line option for configure that lets the user turn this off.

Reply:
Can do.